src/core/compact_object.cc

```diff
@@ -291,7 +297,7 @@ static_assert(sizeof(CompactObj) == 18);
 
 namespace detail {
 
-size_t RobjWrapper::MallocUsed() const {
+size_t RobjWrapper::MallocUsed(bool slow) const {
```

Comment:
This works, but isn't very elegant. Why don't you want to cache this size on the object like we do elsewhere?

Reply:
I am planning to, once we switch to QList. Also with streams it may take some time as well.
Meanwhile I would like to have at least the "slow" variants because performance is not an issue for "memory usage".